specs/driver_spec.rb

```diff
@@ -55,26 +55,137 @@
     end
   end
 
-  describe "average_rating method" do
+  describe "get_average_rating method" do
     before do
       @driver = RideShare::Driver.new(id: 54, name: "Rogers Bartell IV", vin: "1C9EVBRM0YBC564DZ")
-      trip = RideShare::Trip.new({id: 8, driver: @driver, passenger: nil, date: "2016-08-08", rating: 5})
+      trip = RideShare::Trip.new({id: 8, driver: @driver, passenger: nil, start_time: "2016-08-08", end_time: Time.parse("2016-08-10"), cost: 1000, rating: 5})
       @driver.add_trip(trip)
     end
 
     it "returns a float" do
-      @driver.average_rating.must_be_kind_of Float
+      @driver.get_average_rating.must_be_kind_of Float
     end
 
     it "returns a float within range of 1.0 to 5.0" do
-      average = @driver.average_rating
+      average = @driver.get_average_rating
       average.must_be :>=, 1.0
       average.must_be :<=, 5.0
     end
 
     it "returns zero if no trips" do
       driver = RideShare::Driver.new(id: 54, name: "Rogers Bartell IV", vin: "1C9EVBRM0YBC564DZ")
-      driver.average_rating.must_equal 0
+      driver.get_average_rating.must_equal 0
     end
   end
+
+  describe "calculate_total_revenue method" do
+    it "returns total revenue of a driver" do
+      driver_data = {id: 3, name: "Lovelace", vin: "12345678912345678"}
```

Comment:
Other questions that I would like to see these tests ask, for all the wave 1 methods (on both `Driver` and `Passenger`):

- What happens if the driver or passenger has no trips?
- What happens if the driver or passenger has an incomplete trip?